.chloggen/resourceprocessory_system_os_version.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

```suggestion
change_type: 'breaking'
```
This is a breaking change because it enables the addition of a new dimension to metrics out of the box, potentially recreating time series when customers update.

---

I have set `os.version` to not be enabled by default, leaving the `change_type` as `enhancement` should be acceptable.